src/app/pages/poll/poll.component.ts

```diff
@@ -130,7 +141,9 @@ export class PollComponent {
     this.changingState = true;
     if (open) {
       this.poll.startDate = this.currentDate;
-      if (this.poll.endDate) this.poll.endDate = null;
+      if (this.poll.endDate) {
+        this.poll.endDate = null;
+      }
     } else {
       this.poll.endDate = this.currentDate;
```

Comment:
what about `startDate` here?

Reply:
Don't know what you mean?! If you manually open the poll, `startDate` is set to the current date and `endDate` is set to `null`. Should work ...

Reply:
ok, so you just leave `startDate` untouched in the case the poll gets manually closed. Seems fine